assets/scss/_search.scss

```diff
@@ -3,7 +3,7 @@
 .td-search {
     background: transparent;
     position: relative;
-    width: 90%;
```

Comment:
FYI: I think that this 90% was an artefact of the many iterations we did while attempting to redesign the search input box a little while back.